modules/ossm-about-adding-projects-using-label-selectors.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/create-mesh.adoc
+
+:_content-type: CONCEPT
+[id="ossm-about-adding-projects-using-label-selectors_{context}"]
+= About adding projects using label selectors
+
+For cluster-wide deployments, you can use label selectors to add projects to the mesh. Label selectors specified in the `ServiceMeshMemberRoll` resource enable the {SMProductShortName} operator to add or remove namespaces to or from the mesh based on namespace labels. Unlike other standard OpenShift resources that allow you to specify a single label selector, the `ServiceMeshMemberRoll` resource allows you to specify multiple label selectors. If the labels for a namespace match any of the selectors specified in the `ServiceMeshMemberRoll` resource, then the namespace is included in the mesh.
```

Comment:
```suggestion
For cluster-wide deployments, you can use label selectors to add projects to the mesh. Label selectors specified in the `ServiceMeshMemberRoll` resource enable the {SMProductShortName} operator to add or remove namespaces to or from the mesh based on namespace labels. Unlike other standard {product-title} resources that you can use to specify a single label selector, you can use the `ServiceMeshMemberRoll` resource to specify multiple label selectors. If the labels for a namespace match any of the selectors specified in the `ServiceMeshMemberRoll` resource, then the namespace is included in the mesh.
```
- Replace OpenShift with an attribute.
- Per ISG, avoid anthropomorphisms like "allow" and "enable" for inanimate objects: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism

Reply:
Done